core/arch/arm/plat-sam/conf.mk

```diff
@@ -10,10 +10,10 @@ $(call force,CFG_PL310,y)
 $(call force,CFG_PL310_LOCKED,y)
 $(call force,CFG_AT91_MATRIX,y)
 
-CFG_TZDRAM_START ?= 0x30000000
-CFG_TZDRAM_SIZE  ?= 0x800000
+CFG_TZDRAM_START := 0x20000000
+CFG_TZDRAM_SIZE  := 0x800000
```

Comment:
The proper syntax for what you're after is `$(call force,CFG_TZDRAM_START,0x20000000)` (and same for size of course).
https://optee.readthedocs.io/en/latest/building/gits/optee_os.html?highlight=call%20force#the-force-macro

Reply:
Thanks @jforissier , I used `force` and added some comments to explain why there are forced.